### PR TITLE
PAPP-21096: Feature - Use dropdown for "indicator type" param.

### PR DESCRIPTION
**Pull request by paulnawat**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Use Dropdown component for the "indicator type" param. The dropdown will display both itype and threat_type options. Users still need to make sure to select valid option for their selected approval checkbox.

## What is the current behavior? (OPTIONAL)
- Input box is used for the "indicator type" param. Users are required to type the value in the box. There is higher chance of typo here, as there are 160 possible values to type here.

## What is the new behavior? (OPTIONAL)
- Use Dropdown component for the "indicator type" param. Users can filter the dropdown by the input text. Users can then select the option they want. This should reduce typo and also help reduce jumping back to the document page.

## Other information (OPTIONAL)
- Since this is simply UI update to reduce human error and make it easier to use the field "indicator type", this should NOT affect the functionalities of this app at all. This implies that no test should be affected.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: threatstream.json [hunk 1]
```diff
@@ -6224,7 +6224,169 @@
                     "description": "Type of observable to import",
                     "data_type": "string",
                     "order": 1,
-                    "required": true
+                    "required": true,
+                    "value_list": [
```
Comment (paulnawat) [inline threatstream.json:6228]: These options consist of itype options and threat_type options. This corresponds to the README:
image

File: readme.html [hunk 2]
```diff
@@ -136,141 +136,210 @@
 	</ul>
 </p>
 
-<p><b>List of indicator types (itype) for the import_&lt;indicator_type&gt;_observable and the import_observables actions (To be given as input when with_approval parameter is set to False)</b>
```
Comment (paulnawat) [inline readme.html:139]: This change here is simply UI update from bullet point display to multi column display:
Screen Shot 2021-12-03 at 4 27 24 PM

File: threatstream.json [hunk 3]
```diff
@@ -7267,14 +8077,14 @@
                     "description": "Type of observable to import",
                     "data_type": "string",
                     "order": 0,
+                    "required": true,
```
Comment (paulnawat) [inline threatstream.json:8080]: Moving this up for better readability.

File: threatstream.json (hunk 1 above)
Comment (paulnawat) [reply threatstream.json:6228]: With this, users won't run into typo issues, since the options are taken directly from the dropdown.

**Review by brian-phantom — APPROVED** [review]

**Review by joshquerinphantom — APPROVED** [review]

**Pull request merged**